MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -5249,126 +5249,123 @@ public Money calculateCostPerJump(boolean excludeOwnTransports, boolean campaign
         // to transport the force. Smaller dropships are represented by half-dropships.
 
         // If we're transporting more than a company, Overlord analogues are more efficient.
-        if(noMech > 12) {
-            leasedLargeMechDropships = noMech / largeDropshipMechCapacity;
+        if (noMech > 12) {
+            leasedLargeMechDropships = (noMech / (largeDropshipMechCapacity * 1.0));
```

Comment:
Just cast to double, avoid multiplying by 1.0.
```suggestion
            leasedLargeMechDropships = noMech / (double)largeDropshipMechCapacity;
```

Reply:
I just realised why that didn't work originally... I put it on the divisor instead of the dividend...